### PR TITLE
UG: Update addMeeting to use INDEX

### LINKED ISSUE #99: Update addMeeting in UG to use index instead
To addMeeting, we will first have to listClients anyway. Instead of typing out full client name, we will use an index from the filterlist.

### PR DESCRIPTION
**Pull request by Thing1Thing2**
close #99

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -115,14 +115,16 @@ Examples:
 Adds a new meeting to MyInsuRec. 
 DATE should be in DDMMYYYY format and TIME should be in 24-hour format.
 
-Format: `addMeeting n/NAME d/DATE t/TIME`
+Format: `addMeeting i/INDEX d/DATE t/TIME`
```
Comment (sikai00) [inline docs/UserGuide.md:118]: Will also require a description field for this actually

**Review by sikai00 — CHANGES_REQUESTED** [review]
Our meetings now has a description field as well so we need to add it in. Prefix wise maybe can use something like dn/<descriptions> (this is just a suggestion, if you have something better please use that!) Thanks for helping to update the UG

File: docs/UserGuide.md (hunk 1 above)
Comment (Thing1Thing2) [reply docs/UserGuide.md:118]: okay  :+1:

**Comment by Thing1Thing2** [comment]
> ... Prefix wise maybe can use something like dn/ ...

dn/ seems okay (d/ already taken 😞)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -115,14 +115,16 @@ Examples:
...
-* A meeting contains NAME, DATE and TIME.
+* A meeting contains INDEX, DATE, TIME and DESCRIPTION.
+* Adds the client at the specific `INDEX` to the meeting.
+* The index refers to the index number shown in the displayed clients' list.
 
 Examples:
-* `addMeeting n/John Tan d/28092022 t/1400`
-* `addMeeting n/John Tan t/1400 d/28092022`
+* `addMeeting i/1 d/28092022 t/1400`
+* `addMeeting i/1 t/1400 d/28092022`
```
Comment (rexong) [inline docs/UserGuide.md:127]: I think we should add the description in the example as well

**Review by rexong — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (Thing1Thing2) [reply docs/UserGuide.md:127]: ok :+1:

**Review by sikai00 — CHANGES_REQUESTED** [review]
same comment as @rexong, just need that in the example

**Review by rexong — APPROVED** [review]
LGTM

**Review by sikai00 — APPROVED** [review]

**Pull request merged**